.gitignore

```diff
@@ -1,3 +1,4 @@
 /target/
 **/*.rs.bk
-/epics.log
+.env
+/epics.log
```

Comment:
Fix your editor so it leaves a newline at the end of files, it'll make diffs cleaner.